manim/mobject/types/opengl_image_mobject.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+from PIL import Image
+
+from ...constants import *
+from ...utils.bezier import inverse_interpolate
+from ...utils.images import get_full_raster_image_path
+from ...utils.iterables import listify
+from ..opengl_mobject import OpenGLMobject
+
+
+class OpenGLImageMobject(OpenGLMobject):
+    shader_dtype = [
+        ("point", np.float32, (3,)),
+        ("im_coords", np.float32, (2,)),
+        ("opacity", np.float32, (1,)),
+    ]
+    shader_folder = "image"
+
+    def __init__(self, filename, opacity=1, height=4, **kwargs):
+        self.img_height = height
+
+        path = get_full_raster_image_path(filename)
+        self.image = Image.open(path)
+        super().__init__(opacity=opacity, texture_paths={"Texture": path}, **kwargs)
+
+    def init_data(self):
+        self.data = {
+            "points": np.array([UL, DL, UR, DR]),
+            "im_coords": np.array([(0, 0), (0, 1), (1, 0), (1, 1)]),
```

Comment:
```suggestion
            "points": np.array([UL, DL, UR, UR, DR, DL]),
            "im_coords": np.array([(0, 0), (0, 1), (1, 0), (1, 0), (1, 1), (0, 1)]),
```

Reply:
Great catch! That makes the image appear more naturally